src/components/AudioResultsList.vue

```diff
@@ -80,23 +87,15 @@ export default {
             localeCount: count.toLocaleString(this.$i18n.locale),
           })
     },
-    currentPage() {
-      return this.$store.state.audioPage
-    },
-    isFetchingAudiosError() {
-      return this.$store.state.isFetchingError.audios
-    },
-    isFetchingAudios() {
-      return this.$store.state.isFetching.audios
-    },
+
     isFinished() {
-      return this.currentPage >= this.$store.state.pageCount.audios
-    },
-    errorMessage() {
-      return this.$store.state.errorMessage
+      return this.currentPage >= this.$store.state.search.pageCount.audios
     },
   },
   methods: {
+    ...mapActions({
+      fetchMedia: `${SEARCH}/${FETCH_MEDIA}`,
+    }),
```

Comment:
For binding namespaced actions, provide the name of the module as the first argument for the `mapActions` call.

```suggestion
    ...mapActions(SEARCH, [
      FETCH_MEDIA,
    ]),
```

Reply:
Ooh, that's a great suggestion @dhruvkb I didn't know you could do that. Bye string concatenation!

Reply:
In that case we can do
```js
...mapActions(SEARCH, {
  fetchMedia: FETCH_MEDIA
}),
```